src/common/conditional_compilation/README.md

```diff
@@ -0,0 +1,19 @@
+# OpenVINO conditional compilation
+
+OpenVINO conditional compilation(CC) can greatly optimize package binaries size for particular one or multiple models by excluding unnecessary components or code region. There are 2 CC modes for different CC build stages: SELECTIVE_BUILD_ANALYZER and SELECTIVE_BUILD.
```

Comment:
I propose to combine this intro with https://github.com/openvinotoolkit/openvino/blob/master/docs/dev/conditional_compilation.md

Because I see here intersections

Reply:
ok, no problem!

Reply:
done